src/components/Synthetics/TradeboxPoolWarnings/TradeboxPoolWarnings.tsx

```diff
@@ -133,6 +141,55 @@ export const useTradeboxPoolWarnings = (
 
   const showHasBetterOpenFeesWarning = canShowHasBetterExecutionFeesWarning;
 
+  const isFirstOrder = !accountStats || accountStats.closedCount === 0;
+  const marketIndexName = marketInfo ? getMarketIndexName(marketInfo) : "";
+  const marketPoolName = marketInfo ? getMarketPoolName(marketInfo) : "";
+
+  useEffect(() => {
+    if (!marketIndexName || !marketPoolName) {
+      return;
+    }
+
+    if (
+      showHasNoSufficientLiquidityInAnyMarketWarning ||
```

Comment:
Hmm, I think this may trigger extra events when we implement automatic pool switching (which now is on Hold)? Say we will implement a hook which switches pools based on user input, and `showHas...` for a short moment may be true. 

Unfortunately can't see good solution, so if you don't have something robust from top of your head - please skip.